tests/unit/Core/Framework/Webhook/Service/WebhookManagerTest.php

```diff
@@ -176,10 +193,39 @@ public function testWebhooksForLiveVersionOnlyAreIgnoredIfPayloadDoesNotHaveLive
         static::assertEmpty($messages);
     }
 
+    public function testWebhooksForLiveVersionOnlyAreSentIfPayloadDoesNotHaveAnyVersionId(): void
+    {
+        $entityRepository = new StaticEntityRepository([], new CustomerDefinition());
+
+        $event = $entityRepository->create([
+            [
+                'id' => Uuid::randomHex(),
+            ],
+        ], Context::createDefaultContext());
+
+        /** @var EntityWrittenEvent $eventByEntityName */
```

Comment:
please do not use var annotations. Especially in tests you can simply assert the type. also see https://github.com/shopware/shopware/blob/trunk/coding-guidelines/core/writing-code-for-static-analysis.md

Reply:
copy & pasted -> fixed that plus the source as well 👍